modules/osu_migrations_media/migrations/upgrade_d7_media_audio.yml

```diff
@@ -0,0 +1,61 @@
+id: upgrade_d7_media_audio
+#class: Drupal\osu_migrations_media\Plugin\migrate\source\FileByType
```

Comment:
Need this comment?

Reply:
Class requirements in yaml are only if we need a seperate migration class:
[Drupal Migrate Yaml Class Key](https://understanddrupal.com/blog/configuration-options-migration-yaml-definition-files/#class)

Reply:
With that the class key should be removed from the yamls or set to the default `\Drupal\migrate\Plugin\Migration`
I have been setting this previously to the default for out of habit.